uturn/app/uturn.hoon

```diff
@@ -0,0 +1,75 @@
+/-  uturn
+/+  default-agent, dbug
+=,  hmac:crypto
+|%
++$  versioned-state
+    $%  state-0
+    ==
++$  state-0  [%0 server-config=(unit server:uturn) threads=(map @ta [=path])]
++$  card  card:agent:gall
+--
+%-  agent:dbug
+=|  state=state-0
+^-  agent:gall
+=<
+|_  =bowl:gall
++*  this     .
+    default   ~(. (default-agent this %|) bowl)
+    helper    ~(. +> bowl)
+::
+++  on-init
+  `this
+++  on-save
+  ^-  vase
+  !>(state)
+++  on-load  :: on-load:default
+  |=  =vase
+  ^-  (quip card _this)
+  =/  old-state  !<(versioned-state vase)
+  ?-  -.old-state
+    %0  `this(state old-state)
+  ==
+++  on-poke
+  |=  [=mark =vase]
+  ^-  (quip card _this)
+  ?+  mark  (on-poke:default mark vase)
+  %set-server-config
+    =/  new-server-config  !<(server:uturn vase)
+    `this(server-config.state (some new-server-config))
+  ==
+::
+++  on-watch
+  |=  =path
+  ^-  (quip card _this)
+  ?>  (team:title src.bowl our.bowl)
+  ?+  path  (on-watch:default path)
+  [%get-server * ~]
+    =/  server-config  (need server-config.state)
+    =/  secret  secret.server-config
+    =/  epoch  (unt:chrono:userlib now.bowl)
+    =/  user  "{<our.bowl>}-{(scow %uv (sham 2 eny.bowl))}" 
+    =/  credential  (make-credential [ttl=86.400 time=epoch secret=secret user=user url=url.server-config])
```

Comment:
You could make these line up more tidily to improve legibility. Also, the `=/  credential` line was shooting past 80 characters, don't hesitate to go tall-form.  
Note also that adding the faces isn't mandatory. It's fair game to leave the `ttl` one in, because it helps clarify what the magic number is, but the other ones are understandable enough without faces.

```suggestion
      [%get-server @ ~]
    =/  config  (need server-config.state)
    =/  secret  secret.config
    =/  epoch   (unt:chrono:userlib now.bowl)
    =/  user    "{<our.bowl>}-{(scow %uv (sham 2 eny.bowl))}" 
    =/  credential
      %:  make-credential
        ttl=86.400
        epoch
        secret
        user
        url.server-config
      ==
```

Also, does the second part of the path (ie `/get-server/this-part`) not need to be used here somehow? If not, why do we require it? (At the very least, as suggested above, the `*` should be a `@`, because the path type doesn't allow anything but an atom there.)